### PR TITLE
Remove Redundant Tests

### LINKED ISSUE #40: Bug: Async Client Fails e2e on Python 3.7
Works fine on 3.10 for me... but CI is failing with an attribute error. It can only be a python version issue. We should bump it.

### PR DESCRIPTION
**Pull request by bh2smith**
Closes #40 (hopefully)

### CODE REVIEW

**Comment by eliseygusev** [comment]
What is strange about the logs is
` AttributeError: 'AsyncDuneClient' object has no attribute 'close_session'. Did you mean: '_create_session'`
because it is now renamed to disconnect (to be consistent with connect) and e.g. https://github.com/cowprotocol/dune-client/blob/main/dune_client/client_async.py#L69 uses it. yet the test with async with fails...

however, i see that i forgot to rename the function in the tests
https://github.com/cowprotocol/dune-client/blob/main/tests/e2e/test_async_client.py#L45

might be a reason why it fails on CI and work locally when unit tests are run

**Comment by bh2smith** [comment]
Right - I hadn't pulled main. trying again.

**Comment by eliseygusev** [comment]
https://github.com/cowprotocol/dune-client/pull/42
Did a PR with fixes

**Comment by eliseygusev** [comment]
Accidentally closed #42 😅  #43 is the new one without conflicts

File: tests/e2e/test_async_client.py [hunk 1]
```diff
@@ -175,14 +175,14 @@ async def test_invalid_job_id_error(self):
...
+        await dune.disconnect()
 
     async def test_disconnect(self):
         dune = AsyncDuneClient(self.valid_api_key)
         await dune.connect()
         results = (await dune.refresh(self.query)).get_rows()
         self.assertGreater(len(results), 0)
-        await dune.close_session()
+        await dune.disconnect()
         self.assertTrue(cl._session.closed)
```
Comment (eliseygusev) [inline tests/e2e/test_async_client.py:186]: there should be `self.assertTrue(dune._session.closed)`, my bad

but getting `Too many requests in client`  is interesting, not sure whats the best way to approach it

Comment (eliseygusev) [inline tests/e2e/test_async_client.py:186]:
```suggestion
        self.assertTrue(dune._session.closed)
```

Comment (bh2smith) [reply tests/e2e/test_async_client.py:186]: Maybe it's retrying too much I'll try it out tomorrow.

**Comment by bh2smith** [comment]
Alright @eliseygusev - there is something wrong with all of this and I don't have time to keep trying to debug it. I suspect that this client is pinging the URL way more frequently that is specified in the "ping_frequency".

**Comment by eliseygusev** [comment]
But the failing tests are in sync_client. I think this might be the artifact of running a lot of tests at once (since this PR added some more) and Dune rate limits those after certain point. I guess I can try adding sleeps after the requests, this would make tests run longer but lessen probability of getting "Too many requests"

**Comment by bh2smith** [comment]
Right! Then maybe I will put some sleep between tests in there. Good point.

**Comment by eliseygusev** [comment]
Yeah, thats true. I don't mind cutting some of those as well.

**Pull request merged**